ProjectImNotSureWhatItIS/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace ProjectImNotSureWhatItIS
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            User user = new User();
+            Console.WriteLine("Hey, can I ask you a few questions");
+
+            user.FirstName = user.AnswerQuestions("What's your first name");
+            user.LastName = user.AnswerQuestions("What's your last name");
+            user.FavAnimal = user.AnswerQuestions("What's your favorite animal?");
+            user.FavColor = user.AnswerQuestions("What's your favorite color?");
+            user.FavNumber = user.AnswerQuestionWithIntiger("What's your lucky number?");
+            user.NumPets = user.AnswerQuestionWithIntiger2("How many pet's do you have?");
+            user.petNames = user.AnswerwithPetNames("And what are their names?");
+
+            Console.WriteLine($"So you are {user.FirstName} {user.LastName}");
+            Console.WriteLine($" your favorite animal is {user.FavAnimal}");
+            Console.WriteLine($"and your favorite color is {user.FavColor} ");
+            Console.WriteLine($"You have {user.NumPets}");
```

Comment:
This line does not express how many "what" they have, it simply states..how many.